src/mds/schedule/scheduler_helper.cpp

```diff
@@ -379,6 +379,43 @@ void SchedulerHelper::CopySetDistributionInOnlineChunkServer(
         }
     }
 }
+
+void SchedulerHelper::CopySetDistributionInOnlineChunkServer(
```

Comment:
Function naming is inappropriate,  use verbs to name it

Reply:
done